redisson-tomcat/redisson-tomcat-10/src/main/java/org/redisson/tomcat/RedissonSessionManager.java

```diff
@@ -324,7 +324,9 @@ public void onMessage(CharSequence channel, AttributeMessage msg) {
                             
                             if (msg instanceof AttributeUpdateMessage) {
                                 AttributeUpdateMessage m = (AttributeUpdateMessage)msg;
-                                session.superSetAttribute(m.getName(), m.getValue(codecToUse.getMapValueDecoder()), true);
+                                Map<String, Object> attrs = new HashMap<>();
```

Comment:
superSetAttribute method can be removed, as this is the only place where it is used.